db/migrate/20221114215902_hide_user_tips_3_to_5_for_existing_users.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class HideUserTips3To5ForExistingUsers < ActiveRecord::Migration[7.0]
+  def up
+    execute "UPDATE user_options SET seen_popups = seen_popups || '{3, 4, 5}'"
+  end
+
+  def down
+    execute "UPDATE user_options SET seen_popups = array_remove(array_remove(array_remove(seen_popups, 3), 4), 5)"
```

Comment:
Might be cleaner to do nothing in the case of a rollback?

Reply:
I think migrations are rarely reverted, so I do not think it is such a mess if we keep this. I can follow up if you feel strongly about this.